src/games/seabattle/Battle.tsx

```diff
@@ -40,6 +42,9 @@ export class Battle extends React.Component<IBattleProps, IBattleState> {
     ).length === 0;
     if (uniqueMove) {
       this.props.moves.salvo(cell.x, cell.y);
+      if (this.props.isAIGame) {
+        setTimeout(this.props.step, 3000);
```

Comment:
why 3s ? isnt it too much? cant it be 1s ?

Reply:
Salvos blink for 2500ms.  I added 500ms so it seems like the ai "thinks" for half a second.